Program.cs

```diff
@@ -147,86 +154,46 @@ public void Tell()
 
         private void WriteFairytaleToConsole()
         {
-            int currentCharacter = 0;
-            Console.WriteLine("{0} planted a {1} and the {1} grew big, very big.", characters[currentCharacter], plant);
-            Console.WriteLine("The {0} began to pull the {1} out of the ground: he pulled and pulled, but could not pull it out.", characters[currentCharacter], plant);
+            activeCharacters++;
+            mainCharacter.ToPlant(plant);

```

Comment:
The code below looks a bit too complicated to me.

would it be possible to represent the characters as an array, starting with the main character, then plant and grow, then use a single loop to pull and call?

Reply:
I changed the logic of the program and made it simpler.